docker-compose.yml

```diff
@@ -1,20 +1,6 @@
 version: "3.5"
 
 services:
-
-  nginx:
```

Comment:
@faysal-ishtiaq could you clarify why the nginx configuration was removed in this PR? I was expect that we would only need to add traefik configuration in order to automate the processes of setting up a security certificate.

Reply:
technically we could have nginx alongside traefik. But both are proxy manager. So, if we want to setup let's encrypt using traefik and put nginx behind it. The requests will pass through traefik, then nginx and then reach the original django app. Is there any specific use-case so that we must use nginx?

Reply:
Ok, understood. We're all very comfortable with nginx and not used traefik before.

1. Other than automating the security certificate, are there other possible advantages of using traefik instead?
2. Could the security certificate automation be achieved with nginx instead and would there be any disadvantages?

The other configuration I've regularly seen added to nginx config is `XSendFile` (`X-Accel-Redirect) to directly serve files using nginx instead of streaming files through Python.

Reply:
There seems to be no advantage of using Traefik other than an automated security certificate. In fact, traefik is about 15% slower than Nginx.

We could automate security certificate automation using certbot, but that setup would not be as easy as the current setup.

Also, I could not find `XSendFile` equivalent in Traefik.